students/sein/westagram/users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class User(models.Model):
+    user_name     = models.CharField(max_length=30)
+    email         = models.EmailField(max_length=100, unique=True)
+    password      = models.CharField(max_length=300)
+    mobile_number = models.CharField(max_length=20, unique=True)
+    other_info    = models.CharField(max_length=100, blank=True)
```

Comment:
```suggestion
    name     = models.CharField(max_length=30)
    email         = models.EmailField(max_length=100, unique=True)
    password      = models.CharField(max_length=300)
    mobile_number = models.CharField(max_length=20, unique=True)
```
1. user table의 name이므로 user_name이 아닌 name으로 변경해 주세요.
2. other_info 컬럼의 용도가 매무 불문명합니다. 없는 것이 더 나을 것 같습니다.

Reply:
수정사항 반영하였습니다. 확인 부탁드립니다.